pkg/kubeadm/master/manifests.go

```diff
@@ -128,6 +160,24 @@ func etcdVolumeMount() api.VolumeMount {
 	}
 }
 
+// certsVolume exposes host SSL certificates to pod containers.
+func certsVolume(s *kubeadmapi.KubeadmConfig) api.Volume {
+	return api.Volume{
+		Name: "certs",
+		VolumeSource: api.VolumeSource{
+			// TODO make path configurable
+			HostPath: &api.HostPathVolumeSource{Path: "/etc/ssl/certs"},
+		},
+	}
+}
+
+func certsVolumeMount() api.VolumeMount {
```

Comment:
Yeah, I suppose this makes sense. But do this imply that etcd certs have to be in `/etc/ssl/certs`? If so, may be worse adding a check for prefix of the path set by the user, WDYT?